resolwe_bio/tools/make_amplicon_table.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python2
```

Comment:
I would like we start using Python 3 as soon as possible and add the required packages to Python 3. The longer we use Python 2, the harder will be to switch.